README.md

```diff
@@ -2,65 +2,217 @@
 
 ## Overview
 
-You can use this document to add a SignalFx wrapper to your AWS Lambda for C#.
+You can use this document to add a SignalFx wrapper to your AWS Lambda for .NET.
 
-The SignalFx C# Lambda Wrapper wraps around an AWS Lambda C# function handler, which allows metrics to be sent to SignalFx.
+The SignalFx C# Lambda Wrapper wraps around an AWS Lambda .NET function handler, which allows metrics and traces to be sent to SignalFx.
 
-## Step 1: Install via NuGet
+At a high-level, to add a SignalFx .NET Lambda wrapper, you can:
+   * Package the code yourself; or
+   * Use a Lambda layer containing the wrapper, and then attach the layer to a Lambda function.
```

Comment:
It could be helpful to have each of these link to their respective sections.  Directing ppl away from the document to refer to layers docs before that section may not be necessary/intuitive until step four.

Reply:
Agree with that I actually created a story to simplify those steps, it is the backlog.